go/obscuronode/host/hostrunner/test.toml

```diff
@@ -0,0 +1,16 @@
+id = "0x0000000000000000000000000000000000000000"
```

Comment:
Probably doesn't matter in this case but just incase anyone adds other tests here, maybe use non-zero values for the test id and rollup address because of golang's default value shenanigans?

Reply:
I want to keep these aligned with the defaults elsewhere.